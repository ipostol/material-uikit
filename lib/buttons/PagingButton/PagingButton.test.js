'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _PagingButton = require('./PagingButton');

var _PagingButton2 = _interopRequireDefault(_PagingButton);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('PagingButton component', function () {

  it('should render correctly', function () {

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_PagingButton2.default, { page: 1, active: true }));

    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly prev props', function () {

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_PagingButton2.default, { prev: true }));

    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly next props', function () {

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_PagingButton2.default, { next: true }));

    expect(wrapper).toMatchSnapshot();
  });
});