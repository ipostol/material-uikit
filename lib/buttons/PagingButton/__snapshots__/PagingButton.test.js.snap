// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagingButton component should render correctly 1`] = `
<Button
  className="button active"
  element="button"
  mod="secondary"
  round="small"
  size="m"
>
  1
</Button>
`;

exports[`PagingButton component should render correctly next props 1`] = `
<Button
  className="button"
  element="button"
  mod="secondary"
  round="small"
  size="m"
>
  <ArrowLeft />
</Button>
`;

exports[`PagingButton component should render correctly prev props 1`] = `
<Button
  className="button"
  element="button"
  mod="secondary"
  round="small"
  size="m"
>
  <ArrowLeft />
</Button>
`;
